src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -532,7 +528,7 @@ private void testResponseRate() {
     
     private InstructorFeedbacksPage navigateToInstructorFeedbacksPage() {
         
-        AppUrl feedbacksPageUrl = createUrl(Const.ActionURIs.INSTRUCTOR_FEEDBACKS_PAGE).withInstructorId(instructorId);
+        AppUrl feedbacksPageUrl = createUrl(Const.ActionURIs.INSTRUCTOR_FEEDBACKS_PAGE).withUserId(instructorId);
```

Comment:
You might want to exclude this line. I know why this needs to be done, but this shouldn't have anything to do with the issue.
@kanghj this is the reason why this test always fails on the first full run, both in Travis and local.